packages/react/src/themes/teams/colors.ts

```diff
@@ -39,16 +39,16 @@ export const naturalColors: NaturalColors = {
     900: '#237B4B', // siteVariables.green04
   },
   grey: {
-    50: '#FFFFFF', // siteVariables.white
-    100: '#E6E6E6',
-    200: '#CDCCCC',
-    300: '#B8B8B8',
-    400: '#A2A2A2',
-    500: '#8C8C8C',
-    600: '#747373',
-    700: '#5F5E5E',
-    800: '#404040',
-    900: '#252424', // siteVariables.black
+    50: '#FFFFFF', // $app-white
```

Comment:
I would restrain changing the colors in the palette for now, it should be design responsibility. Were all these changed colors not even used?

Reply:
reverted changes in colors, should be ok now